<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- <node name="pos_vel_plot" pkg="rqt_plot" type="rqt_plot" 
    args="/visualizer/speed /visualizer/total_thrust /visualizer/tilt_angle /visualizer/body_rate">
  </node> -->
  
  <node pkg="px4_controller" type="race_track_planning" name="race_track_planning_node" output="screen" >
  	<rosparam file="$(find px4_controller)/config/race_track_planning.yaml" command="load" />
  </node>

</launch>
